src/main/java/uk/gov/hmcts/darts/audio/helper/TransformedMediaHelper.java

```diff
@@ -54,14 +56,21 @@ public UUID saveToStorage(MediaRequestEntity mediaRequest, BinaryData binaryData
 
     @Transactional
     public TransformedMediaEntity createTransformedMediaEntity(MediaRequestEntity mediaRequest, String filename,
-                                                               OffsetDateTime startTime, OffsetDateTime endTime) {
+                                                               OffsetDateTime startTime, OffsetDateTime endTime,
+                                                               Long fileSize) {
+        AudioRequestOutputFormat audioRequestOutputFormat = AudioRequestOutputFormat.MP3;
+        if (mediaRequest.getRequestType().equals(DOWNLOAD)) {
+            audioRequestOutputFormat = AudioRequestOutputFormat.ZIP;
+        }
         TransformedMediaEntity entity = new TransformedMediaEntity();
         entity.setMediaRequest(mediaRequest);
         entity.setOutputFilename(filename);
         entity.setStartTime(startTime);
         entity.setEndTime(endTime);
         entity.setCreatedBy(mediaRequest.getCreatedBy());
         entity.setLastModifiedBy(mediaRequest.getCreatedBy());
+        entity.setOutputFilesize(fileSize.intValue());
```

Comment:
@hemantasharma1129 Wondering, should we be getting @barry-jones50410 to modify the Table "darts.transformed_media" `output_filesize` column type to be similar to Table "darts.media" `file_size`?

Reply:
I will also mention to Hemanta as I was concerned also that int might not be big enough

Reply:
Leaving this column as it is for now as per response from Hemanta